aptos-move/framework/aptos-framework/sources/configs/staking_config.spec.move

```diff
@@ -158,12 +166,14 @@ spec aptos_framework::staking_config {
         use std::signer;
         include StakingRewardsConfigRequirement;
         let addr = signer::address_of(aptos_framework);
-        let staking_reward_config = global<StakingRewardsConfig>(@aptos_framework);
-
         aborts_if addr != @aptos_framework;
-        aborts_if staking_reward_config.rewards_rate_period_in_secs != rewards_rate_period_in_secs;
+        aborts_if global<StakingRewardsConfig>(@aptos_framework).rewards_rate_period_in_secs != rewards_rate_period_in_secs;
         include StakingRewardsConfigValidationAbortsIf;
         aborts_if !exists<StakingRewardsConfig>(addr);
+        ensures global<StakingRewardsConfig>(@aptos_framework).rewards_rate == rewards_rate;
```

Comment:
could you use `let post ...` to remove duplicated global<StakingRewardsConfig>(@aptos_framework)?

Reply:
fixed